mitodl/ocw-hugo-themes
# Wassaf/28 ocw hugo footer

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #28  

#### What's this PR do?

- Updates footer as per invision design 
- Adds footer to Home Page About and Educator Page

#### How should this be manually tested?
- Verify footer is responsive
- Verify all links point to correct URLs
- Verify spacing and font sizes

#### Screenshots (if appropriate)

![Screenshot 2022-01-14 at 3 15 59 AM](https://user-images.githubusercontent.com/87968618/149423412-8dcbb034-9960-4325-bdde-4d104a62333c.png)

![Screenshot 2022-01-14 at 3 15 52 AM](https://user-images.githubusercontent.com/87968618/149423617-f3d00bf0-fc31-4b92-8ac1-c8fc5156c681.png)

![Screenshot 2022-01-14 at 4 16 19 AM](https://user-images.githubusercontent.com/87968618/149423963-cefe1b35-0441-4ea9-9d43-487ea5fada05.png)

![Screenshot 2022-01-14 at 4 16 47 AM](https://user-images.githubusercontent.com/87968618/149423977-66cbb62b-1b74-4197-a366-47a301c4339a.png)

![Screenshot 2022-01-14 at 4 17 01 AM](https://user-images.githubusercontent.com/87968618/149423990-b4f3fa56-3b44-46e4-a5c2-5b8d32787a95.png)

<img width="1424" alt="Screenshot 2022-01-26 at 4 44 59 PM" src="https://user-images.githubusercontent.com/87968618/151157831-9f50db35-9b25-40b9-8eeb-c37f523a53fe.png">
